tests/unit/queryBuilder/QueryBuilder.js

```diff
@@ -729,15 +729,16 @@ describe('QueryBuilder', () => {
   });
 
   it('resultSize should create and execute a query that returns the size of the query', done => {
-    mockKnexQueryResults = [[{ count: 123 }]];
+    mockKnexQueryResults = [[{ count: '123' }]];
     QueryBuilder.forClass(TestModel)
       .where('test', 100)
       .orderBy('order')
+      .limit(10)
+      .offset(100)
       .resultSize()
       .then(res => {
         expect(executedQueries).to.have.length(1);
         expect(res).to.equal(123);
-        // resultSize should cancel the groupBy call since it doesn't affect the outcome.
```

Comment:
Removed this comment because a `GROUP BY` on the underlying query would affect the outcome of a `COUNT`.

Reply:
Yeah it should have said "orderBy" instead.